gympp/include/gympp/Robot.h

```diff
@@ -63,6 +65,16 @@ struct gympp::Limit
     double max = std::numeric_limits<double>::max();
 };
 
+struct gympp::ContactData
+{
+    std::string bodyA;
+    std::string bodyB;
+    std::array<double, 3> depth;
+    std::array<double, 6> wrench;
+    std::array<double, 3> normal;
```

Comment:
Just  to understand, why we add this attributes if they are unused? Would it make more sense to add them when we support them?

Reply:
Data exposed from gazebo won't likely change in the next releases. Right now, even if not all these data are exposed, Gazebo already has them part of its API. In order to reduce our API changes in the future, since in upstream there's already the support of them, I though we could already have them too.